Panic running a v0.0.7 client against a v0.0.6 operator

##### Issue Overview

Panic, presumable caused by Route hint, on ike create

##### Expected Behaviour

Missing feature

##### Current Behaviour

Panic

```
./dist/ike create --deployment reviews-v1 --image quay.io/maistra-dev/istio-workspace-test:v0.0.6-next -n bundle-test
target found	{"Deployment": "reviews-v1-v1-aslak-kuhtr"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14658a2]

goroutine 1 [running]:
github.com/maistra/istio-workspace/pkg/internal/session.CreateOrJoinHandler(0x7ffc367b2343, 0xb, 0x7ffc367b22f8, 0xa, 0xc0003f9e50, 0xb, 0x0, 0x0, 0x1bb3cdc, 0xe, ...)
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/pkg/internal/session/session.go:105 +0x2c2
github.com/maistra/istio-workspace/pkg/cmd/internal/session.Sessions(0xc000507680, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/pkg/cmd/internal/session/session_func.go:32 +0x182
github.com/maistra/istio-workspace/pkg/cmd/create.NewCmd.func2(0xc000507680, 0xc000704ae0, 0x0, 0x6, 0x0, 0x0)
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/pkg/cmd/create/cmd.go:29 +0xa5
github.com/spf13/cobra.(*Command).execute(0xc000507680, 0xc000704a80, 0x6, 0x6, 0xc000507680, 0xc000704a80)
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/vendor/github.com/spf13/cobra/command.go:852 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000507180, 0x1c7ced8, 0x7, 0x7)
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/vendor/github.com/spf13/cobra/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/vendor/github.com/spf13/cobra/command.go:897
main.main()
	/home/aslak/.gvm/pkgsets/go1.15.5/istio/src/github.com/maistra/istio-workspace/cmd/ike/main.go:48 +0x2db

```

##### Steps To Reproduce

1. Install operator v6
2. Use ike create in v7 to create a session

--- a/pkg/internal/session/session.go
+++ b/pkg/internal/session/session.go
@@ -99,10 +99,14 @@ func CreateOrJoinHandler(opts Options, client *Client) (State, func(), error) {
 	if err != nil {
 		return State{}, func() {}, err
 	}
+	route := session.Status.Route
+	if route == nil {
+		route = &istiov1alpha1.Route{}
+	}
 	return State{
 			DeploymentName: serviceName,
 			RefStatus:      getCurrentRef(opts.DeploymentName, *session),
-			Route:          *session.Status.Route,
+			Route:          *route,
 		}, func() {
 			h.removeOrLeaveSession()
 		}, nil

